tests/phpunit/Unit/SkinJsonSkinStylesTest.php

```
@@ -0,0 +1,44 @@
+use MediaWikiUnitTestCase;
+
+/**
+ * Guards skin.json's ResourceModuleSkinStyles against paths that do not
+ * exist on disk. A broken registration is silent in production: the
+ * override simply never ships, or the target module fails to load its
+ * skin styles, and nothing else notices.
+ *
+ * @group Citizen
+ * @coversNothing
+ */
+class SkinJsonSkinStylesTest extends MediaWikiUnitTestCase {
```

PHPStan (2.2.7)

[error] Class MediaWiki\Skins\Citizen\Tests\Unit\SkinJsonSkinStylesTest extends unknown class MediaWikiUnitTestCase.
Learn more at https://phpstan.org/user-guide/discovering-symbols

(class.notFound)